server/src/controller.py

```diff
@@ -22,9 +23,39 @@ def get_slack_users(request=None):
                 formatted_slack_users.append(formatted_slack_user)
         return json(ok(formatted_slack_users))
 
+
+def get_app_users(request, session):
+    users_response = rest_service.app_users(session)
+
+    users = [{
+             'username': user.username,
+             'name': user.name,
+             'score': user.score,
+             'team': user.team
+             } for user in users_response]
+
+    return json(ok(users))
+
+
+def add_user(request, session):
+    # super preliminary
+    request = request.json
+    data = {
+        "username": request["username"],
+        "name": request["name"],  # this should actually be retrieved from slack
```

Comment:
We're not going to have a normal sign up process? Users need to attach their slack?